src/app/components/map/map.component.ts

```diff
@@ -306,10 +306,10 @@ export class MapComponent implements AfterViewInit {
     studentResidencesP.setMap(this.map);
 
     //Text properties for all buildings
-    var markerColor = 'purple';
+    var markerColor = 'black';
     var fontWeight = 'bold';
     var fontSize = '30px';
-    var iconEmpty = '../res/img/empty.png';
+    var iconEmpty = ''//'../res/img/empty.png';
```

Comment:
For the Building Markers text, I showed the red marker behind it.
Not sure it looks better this way or not